Backported fragment from latest iTALCs libvncclient library. A debugged core dump revealed
that client->frameBuffer was NULL and therefore the COPY_RECT created a SEGFAULT.
Have a look at bug #38413 for details.

--- italc-2.0.0.ORIG/ica/x11/libvncclient/rfbproto.c	2011-08-03 23:18:00.000000000 +0200
+++ italc-2.0.0/ica/x11/libvncclient/rfbproto.c	2015-04-30 21:59:35.308228967 +0200
@@ -140,6 +140,10 @@
 static void FillRectangle(rfbClient* client, int x, int y, int w, int h, uint32_t colour) {
   int i,j;
 
+  if (client->frameBuffer == NULL) {
+	return;
+  }
+
 #define FILL_RECT(BPP) \
     for(j=y*client->width;j<(y+h)*client->width;j+=client->width) \
       for(i=x;i<x+w;i++) \
@@ -157,6 +161,10 @@
 static void CopyRectangle(rfbClient* client, uint8_t* buffer, int x, int y, int w, int h) {
   int j;
 
+  if (client->frameBuffer == NULL) {
+	return;
+  }
+
 #define COPY_RECT(BPP) \
   { \
     int rs = w * BPP / 8, rs2 = client->width * BPP / 8; \
